NotificationSystem/NotificationSystem.csproj

```diff
@@ -4,11 +4,12 @@
     <AzureFunctionsVersion>v3</AzureFunctionsVersion>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Azure.Storage.Queues" Version="12.11.0" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.CosmosDB" Version="3.0.5" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.SendGrid" Version="3.0.0" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Storage" Version="4.0.2" />
-    <PackageReference Include="Microsoft.NET.Sdk.Functions" Version="3.0.3" />
+    <PackageReference Include="Azure.Storage.Queues" Version="12.11.1" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.CosmosDB" Version="3.0.10" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.SendGrid" Version="3.0.2" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Storage" Version="4.0.5" />
+    <PackageReference Include="Microsoft.NET.Sdk.Functions" Version="3.1.2" />
+    <PackageReference Include="System.Net.NameResolution" Version="4.3.0" />
```

Comment:
What is this package for?

Reply:
System.Net.NameResolution was getting downgraded. I believe in the old build the downgrade was only a warning or not showing up at all. In the updated pipeline it was causing the pipeline to error out (as it should :-)). The forced update addresses this, but as Edwin works on upgrading to dotnet 6.x, we may just need to remove it. Needed until that is done. I'll let him now directly to keep an eye out for this (I would @mention him, but it seems his github account esolor hasn't been added to the project yet).